pkgs/development/tools/flatpak-builder/default.nix

```diff
@@ -36,23 +36,23 @@
 , libyaml
 , ostree
 , patch
-, python2
 , rpm
 , unzip
+, attr
 }:
 
 let
   installed_testdir = "${placeholder "installedTests"}/libexec/installed-tests/flatpak-builder";
-  installed_test_metadir = "${placeholder "installedTests"}/share/installed-tests/flatpak-builder";
-in stdenv.mkDerivation rec {
+in stdenv.mkDerivation (finalAttrs: {
   pname = "flatpak-builder";
-  version = "1.2.3";
+  version = "1.4.0";
```

Comment:
Could you bump to 1.4.2?

Reply:
I'm currently bumping to 1.4.1 as 1.4.2 seems to be having a problem. I'll take a look later.

Reply:
Bumped.